server/src/index.js

```diff
@@ -41,8 +41,7 @@ async function start() {
   }
 
   const apolloConfig = {
-    typeDefs,
-    resolvers,
+    schema,
```

Comment:
This is the workaround, needs to be reverted and subscriptions added once the voyager bug is fixed.